tags/app.tag

```diff
@@ -1,13 +1,20 @@
 <app>
   <nav><a each={ data } href="#{ id }"> { title }</a></nav>
 
+  <p>
+    <a href="#/">Home</a> |
+    <a href="#/1">create app</a> |
+    <a href="#/2">app detail</a>
+  </p>
```

Comment:
We can use simple `<a>` tags for the routing. Everything after the `#` is a the route